src/cldr/load/webpack.ts

```diff
@@ -31,7 +31,8 @@ export default function loadCldrData(data: CldrData | string[]): P<void> {
 	}
 
 	loadInjectedData();
-	return baseLoad(data);
+	baseLoad(data);
+	return P.resolve();
```

Comment:
Technically it's fine as it was, but the entire reason for moving common functionality into `cldr/load/default` was so they could be correctly pulled in by the webpack build. So I changed `cldr/load/webpack` to import from `cldr/load/webpack`, whose default export does not return a promise. But since the interfaces should be the same across the modules, I will update `cldr/load/default.default` to return a Promise.